tokio-util/src/task/join_map.rs

```diff
@@ -416,8 +416,8 @@ where
     ///  * `None` if the `JoinMap` is empty.
     ///
     /// [`tokio::select!`]: tokio::select
-    pub async fn join_one(&mut self) -> Option<(K, Result<V, JoinError>)> {
-        let (res, id) = match self.tasks.join_one_with_id().await {
+    pub async fn join_next(&mut self) -> Option<(K, Result<V, JoinError>)> {
+        let (res, id) = match self.tasks.join_next_with_id().await {
```

Comment:
It probably doesn't hurt to leave a `#[doc(hidden)]` and deprecated version with the old name. We aren't required to by our stability promises, but it also doesn't hurt to do so. We can remove it when we stabilize `JoinSet`.